contentcuration/contentcuration/frontend/settings/pages/UsingStudio/index.vue

```diff
@@ -64,16 +66,18 @@
     <!-- Issues -->
     <h2>{{ $tr('notableIssues') }}</h2>
     <KExternalLink
+      class="kexternal-redirect"
       href="https://github.com/learningequality/studio/issues/3992"
       :text="$tr('issueLink1')"
-      target="_blank"
+      openInNewTab="true"
       rel="noopener noreferrer"
     />
     <p>{{ $tr('issue1') }}</p>
     <KExternalLink
+      class="kexternal-redirect"
       href="https://github.com/learningequality/studio/issues"
       :text="$tr('issuesPageLink')"
-      target="_blank"
+      openInNewTab="true"
```

Comment:
The` openInNewTab="true"` is used correctly. However, in Vue, boolean props can be passed without a value. So it could be simplified to just` openInNewTab`.
<img width="350" alt="Screenshot 2024-08-06 at 22 37 05" src="https://github.com/user-attachments/assets/52fad4a4-12a7-4489-a906-f251d4cdfee2">

Reply:
Thank you for letting me know. I've changed it to simplified.